src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
+  } else if (days < 7) {
+    result = days * 40 - 20;
+  } else {
+    result = days * 40 - 50;
+  }
+
```

Comment:
lets move all numbers to vars with corresponding naming

Reply:
check the checklist 
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

Reply:
i not see this checklist before. where it was?

Reply:
here it is 
![image](https://user-images.githubusercontent.com/99331793/201110054-b32937c1-a4f4-4b82-b9df-8ddd69fbc72f.png)